src/fondant/dataset/dataset.py

```diff
@@ -857,4 +858,4 @@ def write(
             resources=resources,
             cache=cache,
         )
-        self._apply(operation)
+        return self._apply(operation)
```

Comment:
The only difference to the `.apply` method is that the `write` method does ignore the produces. 

I would leave it like this but what do you think about adding a deprecation warning to the `write` method? 

We could use this method in the future export the dataset from the working directory to a remote storage for instance.

Reply:
I'm not sure I completely understand. Are we deprecating the write ?

Reply:
Thats the question if we want to do it. The apply and write looking quite similar now.